src/content/concepts/mode.md

```diff
@@ -80,3 +80,26 @@ module.exports = {
 -  ]
 }
 ```
+
+If you want to change the behavior according the **mode** variable inside the *webpack.config.js* you have to export a function instead of an object:
+
+```javascript
+var config = {
+  entry: "./app.js"
+  ..
+}
+
+module.exports = (env, argv) => {
+  console.log(argv.mode)        // outputs the mode
```

Comment:
I don't think this line is needed since we already show the `(argv.mode === "development")`  check below which indicates what `argv.mode` is. Anyway, if we're keeping this line, please add the missing semi-colon.

Reply:
I agree, this line can be removed for sure.

Reply:
@m4ss1m0g Please remove it